lib/widgets/avatar/avatar.dart

```diff
@@ -35,22 +35,26 @@ class Avatar extends StatelessWidget {
     final fallbackLetters = name?.getShortcutNameForAvatar() ?? '@';
     return ClipRRect(
       borderRadius: BorderRadius.circular(size / 2),
-      child: MxcImage(
-        key: Key(mxContent.toString()),
-        uri: mxContent,
-        fit: BoxFit.cover,
-        width: size,
-        height: size,
-        cacheKey: mxContent.toString(),
-        placeholder: (context) => RoundAvatar(
-          size: size,
-          text: fallbackLetters,
-          boxShadows: boxShadows,
-          textStyle: TextStyle(
-            fontSize: fontSize,
-            color: textColor ?? AvatarStyle.defaultTextColor(_havePicture),
-            fontFamily: AvatarStyle.fontFamily,
-            fontWeight: AvatarStyle.fontWeight,
+      child: InkWell(
+        onTap: onTap,
+        customBorder: const CircleBorder(),
+        child: MxcImage(
```

Comment:
MxcImage has already an inkwell why not adding an `onTap` parameter that overrides the native one ? Also you can use `rounded: true` for your borders.

IMO it's not good to add clickable area above other clickable areas

Reply:
rounded is just borderRadius 12, not a circle
MxcImage also dont have onTap

Reply:
we have `onTapPreview` if "isPreview" is true but what I mean is we could refactor mxcImage so we can give it an optional onTap if needed and get rid of onTapPreview. Anyway it was just my idea, close the comment if you dont like it ! :)

Reply:
MxcImage should be refactor in onather PR
It impact many screens